mautic/mautic
# Keep session alive in background.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ X ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try creating a email and edit the template.
3. Walk away for at least 24 minutes.
4. Then try to save the changes of the email template.
5. You should be able to save it without losing any edited data.
